domain.yml

```diff
@@ -1,6 +1,6 @@
 session_config:
-  session_expiration_time: 480  # value in minutes, 0 means infinitely long
-  carry_over_slots_to_new_session: true  # set to false to forget slots between sessions
+  session_expiration_time: 480.0
+  carry_over_slots_to_new_session: true
 intents:
 - affirm
 - ask_builder
```

Comment:
Looks like we ddin't change anything to the domain, rasa x did -- let's just remove the diff bc it looks to be screwing up e.g. the retrieval intents

Reply:
urg i undid that so many times and still ...

Reply:
Hahaha. I guess just at the very end 😄do you always `git add .`? you can also `git add folder/` or `git add path/to/filename.ext`

Reply:
no I usually `git add -u` or specific folders/filenames. Since domain had to get new action names I didn't realize other changes also got committed